README.md

```diff
@@ -47,21 +55,29 @@ on:
 jobs:
   release:
     name: 'Zenoo Release'
-    uses: zenoolabs/github-actions/.github/workflows/release.yml@v4
+    uses: zenoolabs/github-actions/.github/workflows/release.yml@v11
     secrets:
       nexus-username: ${{ secrets.NEXUS_USERNAME }}
       nexus-password: ${{ secrets.NEXUS_PASSWORD }}
+      aws-access-key: ${{ secrets.AWS_ACCESS_KEY }}
+      aws-secret-key: ${{ secrets.AWS_SECRET_KEY }}
+    with:
```

Comment:
Shouldn't we add here `force_version` as well which will come from the input of this action?

Reply:
Not needed since it uses the original github event. Deploy action works like that with command